pkg/container-utils/docker/docker.go

```diff
@@ -24,15 +24,15 @@ import (
 	dockertypes "github.com/docker/docker/api/types"
 	dockerfilters "github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/client"
+
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/cgroups"
 	runtimeclient "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/runtime-client"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
-	Name              = "docker"
-	DefaultSocketPath = "/run/docker.sock"
-	DefaultTimeout    = 2 * time.Second
+	Name           = "docker"
+	DefaultTimeout = 2 * time.Second
```

Comment:
I think it is useful for people using only the `container-utils` package to have a default value here.